simtools/db_handler.py

```diff
@@ -492,10 +521,6 @@ def _getSiteParametersYaml(self, site, version, onlyApplicable=False):
         dict containing the parameters
         '''
 
-        if site.lower() not in ['north', 'south']:
-            raise ValueError('Site must be "North" or "South" (not case sensitive)')
-        site = 'lapalma' if 'north' in site.lower() else 'paranal'
```

Comment:
Why is this removed? I think it's needed to read the site variables (they are all in the same file and the only difference is the prefix lapalma/paranal).

Reply:
That was a mistake. I meant to remove the validation only and ended up removing the following line too. I fixed it now.